ci-operator/step-registry/hypershift/dependabot-triage/setup/hypershift-dependabot-triage-setup-commands.sh

```
@@ -7,4 +7,17 @@ echo "=== HyperShift Dependabot Triage Setup ==="
 echo "Verifying Claude Code CLI..."
 claude --version || { echo "ERROR: Claude Code CLI not found"; exit 1; }
 
+# Restore .claude.json from backup if missing (can happen after image rebuilds)
+CLAUDE_CONFIG="${HOME}/.claude/.claude.json"
+if [ ! -f "$CLAUDE_CONFIG" ]; then
+  LATEST_BACKUP=$(ls -t "${HOME}/.claude/backups/"*.backup.* 2>/dev/null | head -1)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)